packages/web3torrent/src/utils/magnet.ts

```diff
@@ -13,28 +12,6 @@ export const useQuery = () => {
   return new URLSearchParams(useLocation().search);
 };
 
-/**
- * Parses the URL from the page, and creates the information necessary in the UI and to start the download.
- * @param infoHash the id of the torrent
- * @param queryParams extra information for the UI, not necessary, but usefull.
- */
-export function parseURL(infoHash: string, queryParams: URLSearchParams): Torrent {
-  if (!infoHash || !infoHash.trim()) {
-    return EmptyTorrent;
-  }
-  const name = queryParams.get('name') || 'unknown';
-  const length = Number(queryParams.get('length')) || 0;
-  const magnetURI = buildMagnetURI(name, length, infoHash);
-  return {...EmptyTorrent, infoHash, name, length, magnetURI};
-}
-
-export function buildMagnetURI(name: string, length: number, infoHash: string): string {
-  return defaultTrackers.reduce(
-    (magnetURI, tracker) => magnetURI + '&tr=' + tracker,
-    `magnet:?xt=urn:btih:${infoHash}&dn=${encodeURIComponent(name)}&xl=${length}`
-  );
-}
-
-export function generateURL({infoHash, length, name}: Torrent): string {
+export function generateURL({infoHash, length, name}: TorrentUI): string {
```

Comment:
this file probably needs a new name, as it generates a URL for a torrent share, it's got nothing to do with a magnet at this point

Reply:
Good call. 1ae612f2c